app/src/main/java/app/revanced/integrations/patches/LithoThemePatch.java

```diff
@@ -19,13 +17,22 @@ public class LithoThemePatch {
         -15790321, // comments chip background (new layout)
         -98492127 // video chapters list background
     };
+    private static final int solidBlack = -16777215;
+
+    private static final int trasparent = 0;
+
+    // boolean used to check if shorts comment box is currently visible
+    public static boolean shortsCommentBox = false;
 
     // Used by app.revanced.patches.youtube.layout.theme.patch.LithoThemePatch
     public static int applyLithoTheme(int originalValue) {
         var isDarkTheme = ThemeHelper.isDarkTheme();
         
-        if ((isDarkTheme && anyEquals(originalValue, DARKCONSTANTS)) || (!isDarkTheme  && anyEquals(originalValue, WHITECONSTANTS)))
-                return 0;
+        if ((isDarkTheme && anyEquals(originalValue, DARKCONSTANTS)))
+            return shortsCommentBox ? solidBlack : trasparent;
+        if (!isDarkTheme && anyEquals(originalValue, WHITECONSTANTS)))
+            return trasparent;
```

Comment:
This gets rid of checking for `isDarkTheme` twice while retaining the code flow logic:

```suggestion
        if (isDarkTheme) {
            if (anyEquals(originalValue, DARKCONSTANTS))
                return shortsCommentBox ? solidBlack : trasparent;
        } else if (anyEquals(originalValue, WHITECONSTANTS))
            return trasparent;
```